LICENSE

```diff
@@ -0,0 +1,201 @@
+      except as required for reasonable and customary use in describing the
+      origin of the Work and reproducing the content of the NOTICE file.
+
+   7. Disclaimer of Warranty. Unless required by applicable law or
+      agreed to in writing, Licensor provides the Work (and each
+      Contributor provides its Contributions) on an "AS IS" BASIS,
+      WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+      implied, including, without limitation, any warranties or conditions
+      of TITLE, NON-INFRINGEMENT, MERCHANTABILITY, or FITNESS FOR A
+      PARTICULAR PURPOSE. You are solely responsible for determining the
+      appropriateness of using or redistributing the Work and assume any
+      risks associated with Your exercise of permissions under this License.
+
+   8. Limitation of Liability. In no event and under no legal theory,
+      whether in tort (including negligence), contract, or otherwise,
+      unless required by applicable law (such as deliberate and grossly
+      negligent acts) or agreed to in writing, shall any Contributor be
+      liable to You for damages, including any direct, indirect, special,
+      incidental, or consequential damages of any character arising as a
+      result of this License or out of the use or inability to use the
+      Work (including but not limited to damages for loss of goodwill,
+      work stoppage, computer failure or malfunction, or any and all
+      other commercial damages or losses), even if such Contributor
+      has been advised of the possibility of such damages.
+
+   9. Accepting Warranty or Additional Liability. While redistributing
+      the Work or Derivative Works thereof, You may choose to offer,
+      and charge a fee for, acceptance of support, warranty, indemnity,
+      or other liability obligations and/or rights consistent with this
+      License. However, in accepting such obligations, You may act only
+      on Your own behalf and on Your sole responsibility, not on behalf
+      of any other Contributor, and only if You agree to indemnify,
+      defend, and hold each Contributor harmless for any liability
+      incurred by, or claims asserted against, such Contributor by reason
+      of your accepting any such warranty or additional liability.
+
+   END OF TERMS AND CONDITIONS
+
+   APPENDIX: How to apply the Apache License to your work.
+
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright [yyyy] [name of copyright owner]
```

Comment:
this should have specify the year and copyright owner.

Reply:
Also, when we bulk update licenses, we should be keeping the oldest copyright year, not the current year.

Reply:
Many projects (Cosmos SDK, Hyperledger / Fabric, Apache Camel) use exactly byte-for-byte the LICENSE file found here: https://www.apache.org/licenses/LICENSE-2.0.txt including the "APPENDIX: How to apply the Apache License to your work." which has that "to be filled in" Copyright notice.  The point is that APPENDIX comes after "END OF TERMS AND CONDITIONS", and your actual copyright notice(s) are supposed to be in the NOTICES file.

I'm inclined to leave it byte-for-byte exactly what is on the apache.org website.

Reply:
@turadg re: "Proposing that we remove NOTICE files, not required by Apache License." the way the license is structured, I don't see anything on the Apache website that says that the copyright should go into the LICENSE file.  That APPENDIX certainly does not.  

One *could* put it there in the LICENSE file, and some projects do.  However, my gut is to leave LICENSE file as written and vetted by their lawyers.

The NOTICE file is seems like a pretty logical place to put the copyright, as Apache puts it there for their projects, and other projects have followed suit.

We could also assert the Copyright and license in our README.md.

Reply:
@kriskowal In my survey documented in #4910, I did not see any projects that had LICENSE (or NOTICE) below the top level, so I don't see copying it down the tree and then having to keep it up to date.

Reply:
Oh, I see. This part of LICENSE is just instructions on how to use it. 👍 removing by block.

The actual copyright holder is specified in NOTICE. This is unlike MIT License in which both are in one file:
https://github.com/facebook/react/blob/main/LICENSE

Fwiw here's an example of a LICENSE that roles up the copyright portion redundant with its NOTICE: https://github.com/moby/moby/blob/master/LICENSE
https://github.com/moby/moby/blob/master/NOTICE